ord_schema/visualization/filters.py

```diff
@@ -660,12 +660,24 @@ def _get_compact_components(
             yield compound, False
 
 
+def _get_compact_products(
+    products: Iterable[reaction_pb2.ProductCompound]
+) -> List[reaction_pb2.ProductCompound]:
+    """Returns a list of product compounds for 'compact' visualization."""
+    roles_to_keep = [reaction_pb2.ReactionRole.PRODUCT]
```

Comment:
Can you add UNSPECIFIED as a role to keep?

Reply:
Done for inputs and outputs.